source/agora/common/Set.d

```diff
@@ -222,3 +228,10 @@ unittest
     arr.dropIndex(1);
     assert(arr == [1, 3]);
 }
+
+unittest
+{
+    auto set = Set!uint.from([2, 4, 6, 8]);
+    auto _2xset = set[].map!(elem => tuple(elem, 2*elem));
```

Comment:
Do we need to use `_` for the name of a local variable?

Reply:
No, its because variable names cant start with numbers and I am lazy :)